Framework/DataHandling/inc/MantidDataHandling/ISISJournal.h

```diff
@@ -0,0 +1,44 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "MantidKernel/System.h"
+
+#include <map>
+#include <string>
+#include <vector>
+
+namespace Mantid {
+namespace DataHandling {
+namespace ISISJournal {
+/**
+  Defines functions to aid in fetching ISIS specific run information from
+  journal files
+ */
+
+using ISISJournalTags = std::vector<std::string>;
+using ISISJournalFilters = std::map<std::string, std::string>;
+using ISISJournalData = std::map<std::string, std::string>;
```

Comment:
Thinking about it I'm not sure of the value of using structs here. `Tags` is just a list of names and the others are name->value pairs which I can't see expanding. `Filters` and `Data` are essentially the same though so I'll perhaps rename these to `RunData`. I also don't like the name `Tags`.... maybe `ValueNames` is better...

Reply:
For the name->value pairs are the names fixed? Or are the `ISISJournalTags` used as the keys in `ISISJournalData`?

Reply:
Yes, the Tags are used as keys in the map and they can be anything so it's extensible that wya. I've just pushed some changes which get rid of the "tags" variable naming altogether because I think this just introduces confusion. I'm just refering to values and value-names now.